### PR TITLE
Missing bug fixes for checklists

### PR DESCRIPTION
**Pull request by synixebrett**
These fixes were not pushed before you merged the PR

### CODE REVIEW

**Comment by mgkid3310** [comment]
I've already added ace_units check in master branch, so you can remove that part. Other two are valid.

**Comment by mgkid3310** [comment]
ps. how's `toFixed 0` different from `round`?

**Comment by synixebrett** [comment]
`round` rounds it, but the type stays a number. Meaning you often need to do `str round`. `toFixed` returns a string so doing `toFixed 0` is faster since it is one command instead of two

File: addons/cockpit/functions/fnc_processSpeed.sqf [hunk 1]
```diff
@@ -15,8 +15,8 @@ if (EGVAR(main,hasACEUnits)) exitWith {
 };
 
 private _output = switch (GVAR(checklistUnits)) do {
-	case ("KIAS"): {_kias + " KIAS"};
-	case ("KM/H"): {_kph + " KM/H"};
+  case "KIAS": { (_kias toFixed 0) + " KIAS" };
+  case "KM/H": { (_kph toFixed 0) + " KM/H" };
```
Comment (mgkid3310) [inline addons/cockpit/functions/fnc_processSpeed.sqf]: Any reason for 2-space indentation & spaces within code block?

File: addons/main/XEH_preInit.sqf [hunk 2]
```diff
@@ -11,6 +11,7 @@ GVAR(hasACEInteractMenu) = isClass (configFile >> "CfgPatches" >> "ace_interact_
 GVAR(hasACEMap) = isClass (configFile >> "CfgPatches" >> "ace_map");
 GVAR(hasACEUnits) = isClass (configFile >> "CfgPatches" >> "ace_units");
 GVAR(hasACEWeather) = isClass (configFile >> "CfgPatches" >> "ace_weather");
+GVAR(hasACEUnits) = isClass (configFile >> "CfgPatches" >> "ace_units");
```
Comment (mgkid3310) [inline addons/main/XEH_preInit.sqf]: Already has ace_units check in line 12

File: addons/cockpit/functions/fnc_processSpeed.sqf (hunk 1 above)
Comment (mgkid3310) [reply addons/cockpit/functions/fnc_processSpeed.sqf]: I wish to have coding style unified with other files.
e.i. https://github.com/mgkid3310/AWESome/blob/96df7be770d8a36d2408b0ec1b24bcd4c00573c4/addons/gpws/functions/fnc_flightPhaseCheck.sqf#L14

**Pull request merged**